src/lib/Tests/EventListener/SystemInfoTabGroupListenerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\EventListener;
+
+use EzSystems\EzPlatformAdminUi\EventListener\SystemInfoTabGroupListener;
+use EzSystems\EzPlatformAdminUi\Tab\SystemInfo\SystemInfoTab;
+use EzSystems\EzPlatformAdminUi\Tab\TabGroup;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\Tab\Event\TabEvents;
+use EzSystems\EzPlatformAdminUi\Tab\Event\TabGroupEvent;
+use EzSystems\EzPlatformAdminUi\Tab\SystemInfo\TabFactory;
+use EzSystems\EzPlatformAdminUi\Tab\TabRegistry;
+use EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry;
+
+class SystemInfoTabGroupListenerTest extends TestCase
+{
+    /**
+     * @var Request
+     */
+    private $request;
+
+    /**
+     * @var TabGroupEvent
+     */
+    private $event;
+
+    /**
+     * @var HttpKernelInterface|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Do we use PHPUnit 6.x? I believe it brings the support for namespaces.

Reply:
yes, please aim for PHPUNit 5.7+ support and use the namespeaces so we don't have to change it later. All other repos are current undergoing lots of changes for this..

Reply:
ok, I will change this.